app/views/govuk_publishing_components/components/docs/checkboxes.yml

```diff
@@ -288,6 +288,23 @@ examples:
         - label: "Other"
           value: "other"
           conditional: "See no styling."
+  checkbox_items_with_conditional_reveal_checked:
+    data:
+      name: "contacting-checked"
+      id: "contacting-checked"
+      heading: "How would you like to be contacted?"
+      hint_text: "Please select all options that are relevant to you."
+      items:
+        - label: "Email"
+          value: "email"
+          conditional: <div class="govuk-form-group"><label class="govuk-label" for="contact-by-email">Email address</label><input class="govuk-input govuk-!-width-one-third" id="contact-by-email" name="contactingme" type="email"></div>
+          checked: true
+        - label: "Phone"
+          value: "phone"
+          conditional: <div class="govuk-form-group"><label class="govuk-label" for="contact-by-phone">Phone number</label><input class="govuk-input govuk-!-width-one-third" id="contact-by-phone" name="contactingme" type="tel"></div>
+        - label: "Other"
+          value: "other"
+          conditional: "See no styling."
```

Comment:
I don't understand this conditional - why "See no styling."?

Reply:
Ah, I copied only this over from the example above and added a `checked: true`. I can look for a decent example.

Reply:
I mean, it's definitely not a blocker - I just wondered why a person want to show 'See no styling' 😕. Don't worry about another example 😄

Reply:
🤷‍♂ Ideally, they should all be real-life examples, since we introduce each feature with a certain need – but we got skittles examples instead 🙂

Reply:
You mean `which-skittles-do-you-like.service.gov.uk` doesn't exist? 🤯

Reply:
Replaced that with contact by text via mobile phone 😁

Reply:
I think `See no styling` is meant to highlight the fact that the component isn't meant to provide styles for the content of the conditional element - that's left to the developer.

Reply:
Ah, that makes sense - thanks!